src/Servers/Kestrel/Kestrel.slnf

```diff
@@ -2,19 +2,38 @@
   "solution": {
     "path": "..\\..\\..\\AspNetCore.sln",
     "projects": [
+      "src\\DataProtection\\Abstractions\\src\\Microsoft.AspNetCore.DataProtection.Abstractions.csproj",
```

Comment:
All the changes in slnf are from HttpStress project dependencies. That project was added to this slnf in another PR: https://github.com/dotnet/aspnetcore/pull/42030

cc @halter73